asip_v2/archive.py

```diff
@@ -281,29 +281,30 @@ def read_icechart_coding(self, fil, filename):
 
     @staticmethod
     def get_the_mask_of_sar_size_data(sar_names, fil, distance_threshold):
+        mask_sar_size = False
```

Comment:
Can you initiate a full size array instead of using `shrink=False`? That will be more understandable.
```suggestion
        mask_sar_size = np.zeros(fil[sar_names[0]][:].shape, bool)
```

Reply:
Also `np.ma.mask_or` ca be replaced with `+`.
mask_sar_size += np.ma.getmaskarray(fil[str_][:])
mask_sar_size += ~np.isfinite(fil[str_])

Although `+=` is less understandable, so this is up to you.

Reply:
mask_or is better than "+". respectfully, let it be as it is.